profiler/src/main/java/com/splunk/opentelemetry/profiler/snapshot/Volume.java

```diff
@@ -23,7 +23,8 @@
 
 public enum Volume implements ImplicitContextKeyed {
   OFF,
-  HIGHEST;
+  HIGHEST,
+  UNSPECIFIED;
```

Comment:
I don't see the volume is part of the spec https://github.com/signalfx/gdi-specification/blob/main/specification/behaviors.md That should be added as this needs to function the same across all agents. UNSPECIFIED is not part of spec as this is internal behavior to agent (it will never be sent over the wire) would be good to add a comment to that effect.